lib/hades_web/auth_pipeline.ex

```diff
@@ -0,0 +1,9 @@
+defmodule HadesWeb.AuthPipeline do
+  use Guardian.Plug.Pipeline, otp_app: :hades,
+                              module: Hades.Guardian,
+                              error_handler: HadesWeb.AuthErrorHandler
+
+    plug Guardian.Plug.VerifyHeader, claims: %{"typ" => "access"}, realm: "Bearer"
```

Comment:
Here we are explicitly saying which kind of token Guardian should verify on header.